sonar-csharp-plugin/src/main/java/org/sonar/plugins/csharp/CSharpCodeCoverageProvider.java

```diff
@@ -152,7 +152,7 @@ public CSharpIntegrationCoverageAggregator(Configuration configuration) {
   public static class CSharpIntegrationCoverageReportImportSensor extends CoverageReportImportSensor {
 
     public CSharpIntegrationCoverageReportImportSensor(CSharpIntegrationCoverageAggregator coverageAggregator,
-                                                       SonarQubeVersion sonarQubeVersion) {
```

Comment:
I may need to ask you about that because the default configuration of my IntelliJ is always indenting params like this,